app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -1,14 +1,157 @@
-import { PageHeading } from '@18f/identity-components';
-import { FormStepsButton } from '@18f/identity-form-steps';
+import { useState, useEffect, useCallback, useRef } from 'react';
 import { useI18n } from '@18f/identity-react-i18n';
+import { PageHeading, LocationCollectionItem, LocationCollection } from '@18f/identity-components';
+
+interface PostOffice {
+  address: string;
+  city: string;
+  name: string;
+  phone: string;
+  saturday_hours: string;
+  state: string;
+  sunday_hours: string;
+  weekday_hours: string;
+  zip_code_4: string;
+  zip_code_5: string;
+}
+
+interface FormattedLocation {
+  addressLine2: string;
+  id: number;
+  name: string;
+  phone: string;
+  saturdayHours: string;
+  streetAddress: string;
+  sundayHours: string;
+  weekdayHours: string;
+}
+
+const locationUrl = '/verify/in_person/usps_locations';
+
+const getResponse = async () => {
+  const response = await fetch(locationUrl).then((res) =>
+    res.json().catch((error) => {
+      throw error;
+    }),
+  );
+  return response;
+};
+
+const formatLocation = (postOffices: PostOffice[]) => {
+  const formattedLocations = [] as FormattedLocation[];
+  postOffices.forEach((po: PostOffice, index) => {
+    const location = {
+      addressLine2: `${po.city}, ${po.state}, ${po.zip_code_5}-${po.zip_code_4}`,
```

Comment:
I would maybe call this `formattedCityStateZip` or something? When I see `addressLine` as a property I always expect things like "Apt #123" or something

Reply:
This will be addressed with LG-7025

Reply:
I decided to address this in this PR, and did so in f51cf5eacaebd32c5d2ecdf1309e6ce9a2a73d00